exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Test for black territory on 5x5 board",
+      "property": "territory",
+      "input": {
+        "board": [
+		    "  B  ",
+		    " B B ",
+		    "B W B",
+		    " W W ",
+		    "  W  "
+		],
+        "x": 0,
+        "y": 1
+      },
+      "expected": {
+        "stone": BLACK
```

Comment:
A couple notes on syntax:

- Values must be one of: string (with quotes), number, array (`[...]`), object (`{ ... }`)
- Values must be followed by a comma unless it is the last value in the current array/object